smcli/main.cpp

```diff
@@ -40,6 +44,26 @@ int main(int argc, char *argv[])
             break;
         }
     }
+    //smos::smcore::SMPReader smpReader;
+    //if (!smpReader.Open("d:\\SourceMonitor\\SourceMonitorOS\\test2.smp"))
+    //return 1;
 
-    return a.exec();
+    //QStringList strList();
+    QDirIterator it("d:\\SourceMonitor\\SM-Test-Files", QStringList() << "*.smp", QDir::Files, QDirIterator::Subdirectories);
```

Comment:
Is it posible to have in the next version a non-hardcoded path for the test files?